confd/templates/docker-compose/docker-compose.tmpl

```diff
@@ -441,9 +446,34 @@ services:
          - northbound.pendev
     mem_limit: ${NB_MEM_LIMIT:-1073741824}
 
+  northbound-blue-green:
```

Comment:
northbound-blue-green is always green. Maybe I should rename it.
yes we still need northbound to run Kilda as usual. Maybe it can be renamed to northbound-blue